bin/commands/init/generate/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+// This program and the accompanying materials are made available
+// under the terms of the Eclipse Public License v2.0 which
+// accompanies this distribution, and is available at
+// https://www.eclipse.org/legal/epl-v20.html
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as os from "cm_os";
+import * as xplatform from "xplatform";
+import * as fs from '../../../libs/fs';
+import * as config from '../../../libs/config';
+import * as common from '../../../libs/common';
+import * as zosFs from '../../../libs/zos-fs';
+import * as zosJes from '../../../libs/zos-jes';
+
+export function execute(dryRun?: boolean) {
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+  const tempFile = fs.createTmpFile();
+  zosFs.copyMvsToUss(ZOWE_CONFIG.zowe.setup.dataset.prefix + '.SZWESAMP(ZWEGENER)', tempFile);
+  let jclContents = xplatform.loadFileUTF8(tempFile, xplatform.AUTO_DETECT);
+
+  jclContents = jclContents.replace(/\{zowe\.setup\.dataset\.prefix\}/gi, ZOWE_CONFIG.zowe.setup.dataset.prefix);
+  jclContents = jclContents.replace(/\{zowe\.runtimeDirectory\}/gi, ZOWE_CONFIG.zowe.runtimeDirectory);
+  let originalConfig = std.getenv('ZWE_PRIVATE_CONFIG_ORIG');
+  let fileIndex = originalConfig.indexOf('FILE(');
+  let lastIndex = 0;
+  let absConfig = '';
+  while (fileIndex != -1) {
+    absConfig += originalConfig.substring(lastIndex, fileIndex+5);
+    let parenIndex = originalConfig.indexOf(')', fileIndex+5);
+    let fileRef = originalConfig.substring(fileIndex+5, parenIndex);
+    let absRef = fs.convertToAbsolutePath(fileRef);
+    absConfig += absRef + ')';
+    lastIndex = parenIndex+1;
+    fileIndex = originalConfig.indexOf('FILE(', lastIndex);
+  }
+  absConfig += originalConfig.substring(lastIndex);
+
+  jclContents = jclContents.replace('FILE <full path to zowe.yaml file>', 'FILE '+absConfig);
```

Comment:
`bin/zwe init generate -c ./yaml/generate.yaml --dry-run`
File path should be absolute:
```
//MYCONFIG DD   *,DLM=$$
FILE ./yaml/generate.yaml
```
Is this correct?
`bin/zwe init generate -c "FILE(./yaml/generate.yaml)" --dry-run`
```
//MYCONFIG DD   *,DLM=$$
FILE FILE(/u/martin/git/qa/jcl/yaml/generate.yaml)
```

Reply:
i think i fixed this now.